packages/proton-bridge/replace_imap-specialuse.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/go.mod b/go.mod
+index fbefaa6..8ce4bae 100644
+--- a/go.mod
++++ b/go.mod
+@@ -77,6 +77,7 @@ replace (
+ 	github.com/emersion/go-imap => github.com/ProtonMail/go-imap v0.0.0-20200828124548-d04b0dc1f399
+ 	github.com/emersion/go-mbox => github.com/ProtonMail/mbox v0.0.0-20200918064939-909a18c9af45
+ 	github.com/emersion/go-smtp => github.com/ProtonMail/go-smtp v0.0.0-20181206232543-8261df20d309
++	github.com/emersion/go-imap-specialuse => github.com/exander77/go-imap-specialuse noannouncecapability
```

Comment:
What is this?

Reply:
This replaces go-imap-specialuse with patched version to solve another IMAP specs issue, already reported here: https://github.com/emersion/go-imap/issues/357

It may be rewritten to a live patch into go workspace libraries but would need some additional code.